src/EnergyPlus/Psychrometrics.hh

```diff
@@ -1292,23 +1292,42 @@ namespace Psychrometrics {
-                                             Real64 const dW1   // humidity ratio  at state 1
+                                             Real64 const W2,   // humidity ratio at state 2
+                                             Real64 const TDB1, // dry-bulb temperature at state 1 {C}
+                                             Real64 const W1    // humidity ratio at state 1
     )
     {
-        // returns sensible enthalpy difference of moist air going from state 1 to state 2
-        Real64 dWavg = 0.5 * (max(dW2, 1.0e-5) + max(dW1, 1.0e-5));
-        return (1.00484e3 + dWavg * 1.85895e3) * (TDB2 - TDB1);
+        // returns sensible enthalpy difference of moist air going from state 1 to state 2 (e.g across coils)
+        // using the minimum humidity ratio state points 1 and 2. This enthalpy difference multiplied by
+        // supply air mass flow rate yields sensible heat transfer rate across coils in Watts
+
+        // the following two functions for calculating enthalpy difference are equivalent:
+        // PsyDeltaHSenFnTdb2W2Tdb1W1() = PsyHFnTdbW(TDB2, min(W1, W2)) - PsyHFnTdbW(TDB1, min(W1,W2))
+        // PsyDeltaHSenFnTdb2W2Tdb1W1() function was derived by simplyfying the above expression
+        // The constant coefficients came from the equation for moist air enthalpy, PsyHFnTdbW()
+
+        Real64 Wmin = min(W1, W2);
+
+        return (1.00484e3 + max(1.0e-5, Wmin) * 1.85895e3) * (TDB2 - TDB1);
     }
 
-    inline Real64 PsyHfgAvgFnTdb2Tdb1(Real64 const TDB2, // dry-bulb temperature at  at state 2 {C}
-                                      Real64 const TDB1  // dry-bulb temperature at  at state 1 {C}
+    inline Real64 PsyDeltaHSenFnTdbEquipTdbWZone(Real64 const TDBEquip, // dry-bulb temperature at equipment outlet {C}
+                                                 Real64 const TDBZone,  // dry-bulb temperature at zone air node {C}
+                                                 Real64 const WZone     // humidity ratio at zone air node
     )
     {
-        // calculate average latent heat of vaporization of water vapor in moist air
-        return (2.50094e6 + 0.5 * (TDB2 + TDB1) * 1.85895e3);
+        // returns sensible enthalpy difference between equipment supply air and zone air evaluated
+        // using the humidity ratio at zone air node. This enthalpy difference multiplied by supply
+        // air mass flow rate yields the sensible heat transfer rate in Watts.
+
+        // the following two functions for calculating enthalpy difference are equivalent:
+        // PsyDeltaHSenFnTdbEquipTdbWZone() = PsyHFnTdbW(TDB2, WZone) - PsyHFnTdbW(TDB1, WZone)
+        // PsyDeltaHSenFnTdbEquipTdbWZone() function was derived by simplyfying the expression above
+        // The constant coefficients come from the equation for moist air enthalpy, PsyHFnTdbW()
+
+        return (1.00484e3 + max(1.0e-5, WZone) * 1.85895e3) * (TDBEquip - TDBZone);
```

Comment:
Can you rewire `PsyDeltaHSenFnTdb2W2Tdb1W1` to call this function?  That way we don't have a copy of these coefficients lying around to maintain?

Reply:
Done. I will run the unit tests suite and push the branch shortly.